cli/running/watchdog.go

```diff
@@ -35,10 +35,12 @@ type Watchdog struct {
 	// restartTimeout describes the timeout between
 	// restarting the Instance.
 	restartTimeout time.Duration
-	// stopped indicates whether Watchdog was stopped.
-	stopped bool
-	// stopMutex used to avoid a race condition under stopped field.
-	stopMutex sync.Mutex
+	// watchdogStateMutex used to avoid a race condition under started and shouldStop fields.
+	watchdogStateMutex sync.Mutex
+	// started indicates whether Watchdog has started an Instance.
+	started bool
```

Comment:
This information can be get from the instance object.

Reply:
Totally agree, fixed.

Reply:
Fixed.